Source/WebCore/html/HTMLElement.cpp

```diff
@@ -1098,6 +1098,9 @@ ExceptionOr<void> HTMLElement::showPopoverInternal(const HTMLElement* invoker)
     if (popoverData())
         popoverData()->setInvoker(invoker);
 
+    if (invoker)
+        invoker->setInvokedPopover(this);
```

Comment:
I feel like this should be called from `PopoverData::setInvoker` and that method should be moved to the C++ file.

Reply:
I guess we can update setInvoker() to take both the popover and the invoker to do this?

Reply:
I've moved it in there, hopefully it's as you'd like.